components/sensor/xiaomi_ble.rst

```diff
@@ -341,11 +341,13 @@ Obtaining The Bindkey
 ---------------------
 
 To set up an encrypted device such as the LYWSD03MMC or CGD1, you first need to obain the bind key. The ``xiaomi_ble`` sensor component is not able to automatically generate a bind key, so you need a different option. 
+The easiest method (confirmed to work for CGD1 but should work for every device) is to use this MiHome modified app: https://www.kapiba.ru/2017/11/mi-home.html
+create folder /your_interlal_storage/vevs/logs/ and pair your BLE device - file pairings.txt with a corresponding data will be created.
```

Comment:
This is a non-English website and thus we cannot add that to this page. As a project we try to be inclusive and accessible for everybody, a website written in Russian doesn't match that.

Therefore, I'm going to friendly decline this contribution 👍 

../Frenck